### PR TITLE
Fix #238

### LINKED ISSUE #238: [1.7.10]Crash when searching item in Custom Stuff 3
Here is the crash report.
https://paste.ee/p/rUyeR

### PR DESCRIPTION
**Pull request by SeraphJACK**
Fix #238

### CODE REVIEW

**Comment by SeraphJACK** [comment]
Add a few null-protections to fix #238 
I've already tested with @SihenZhang that it'll be fixed.

**Comment by KorDum** [comment]
Please use codestyle and formatting from https://github.com/ForestryMC/Binnie/issues/123

File: src/main/java/binnie/genetics/item/ItemSerum.java [hunk 1]
```diff
@@ -93,7 +93,9 @@ public GeneItem getGeneItem(ItemStack stack) {
 	public String getItemStackDisplayName(ItemStack stack) {
 		IGeneItem gene = getGeneItem(stack);
 		BreedingSystem breedingSystem = Binnie.Genetics.getSystem(gene.getSpeciesRoot());
-		return I18N.localise("genetics.item.serum.name", breedingSystem.getDescriptor());
+		if(breedingSystem != null)
+			return I18N.localise("genetics.item.serum.name", breedingSystem.getDescriptor());
+		return I18N.localise("genetics.item.serum.name");
```
Comment (KorDum) [inline src/main/java/binnie/genetics/item/ItemSerum.java:98]: It works without formatted args?

**Review by KorDum — CHANGES_REQUESTED** [review]

File: src/main/java/binnie/genetics/item/ItemSerum.java (hunk 1 above)
Comment (KorDum) [reply src/main/java/binnie/genetics/item/ItemSerum.java:98]: `genetics.item.serum.name=%s Serum`

**Pull request closed without merging**